docs/reference/repositories-metering/repositories-metering.asciidoc

```diff
@@ -0,0 +1,15 @@
+[role="xpack"]
+[testenv="enterprise"]
+[[repositories-metering-apis]]
+== Repositories metering APIs
+
+experimental[]
+
+You can use the following APIs to retrieve repositories metrics.
+This is an API meant to be used by Elastic's commercial offerings.
```

Comment:
I think it would be good to add a few words on how this API is supposed to be used, even if it's just for our commercial offerings